src/v/test_utils/registered_urls.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+
+#include "seastarx.h"
+
+#include <seastar/core/sstring.hh>
+#include <seastar/http/httpd.hh>
+
+#include <iosfwd>
+#include <unordered_map>
+
+namespace http_test_utils {
+struct response {
+    ss::sstring body;
+    ss::httpd::reply::status_type status;
+};
+
+std::ostream& operator<<(std::ostream& os, const response& resp);
+
+struct registered_urls {
+    using content_reply_map = std::unordered_map<ss::sstring, response>;
```

Comment:
we usually use `absl::flat_hash_map` instead of the std variants in the codebase

Reply:
fixed